### DISCUSSION TITLE
[Icon] Medium size replaced with Medium brand icon

### DISCUSSION THREAD

**Discussion by AdrienLZ** [fomantic/Fomantic-UI#1331] [Elements (button, image, label, loader, ...)] [answered]
You can't size an icon with medium size because it display the "medium" brand icon instead of changing the size.

```html
<i class="large tag icon"></i> <!-- Display the tag icon with large size -->
<i class="medium tag icon"></i> <!-- Display the medium icon with default size -->
```

Example: https://jsfiddle.net/s5wqvbd6/

**Comment by lubber-de** [comment, marked as answer]
There does no `medium` size exist for icon.
medium size is considered default without any size classes
See https://jsfiddle.net/rLwhxgef/
